packages/array-max/src/index.js

```diff
@@ -3,39 +3,46 @@ import isArray from 'is-any-array';
 /**
  * Computes the maximum of the given values
  * @param {Array<number>} input
- * @param {int} [options.from] - Start index (inclusive) for the slice within which we look for the max
- * @param {int} [options.to] - End index (exclusive) for the slice within which we look for the max
+ * @param {number} [options.fromIndex] - Start index (inclusive) for the slice within which we look for the max
+ * @param {number} [options.toIndex] - End index (exclusive) for the slice within which we look for the max
  * @return {number}
  */
 export default function max(input, options = {}) {
-  let { from, to } = options;
-
   if (!isArray(input)) {
     throw new TypeError('input must be an array');
   }
 
   if (input.length === 0) {
     throw new TypeError('input must not be empty');
   }
+  let { fromIndex, toIndex } = options;
 
-  if (from === undefined) {
-    from = 0;
-  } else if ((from < 0) | (from > input.length)) {
+  if (fromIndex === undefined) {
+    fromIndex = 0;
+  } else if (
+    (fromIndex < 0) |
+    (fromIndex > input.length) |
```

Comment:
```suggestion
    (fromIndex >= input.length) ||
```

fromIndex cannot be === input.length because that would leave no room for toIndex